docs/PROI/.gitignore

```diff
@@ -24,10 +24,6 @@
 # these rules might exclude image files for figures etc.
 # *.ps
 # *.eps
-*.pdf
-
```

Comment:
@tcoro Why did you remove that? PDF file can are generated from some code (in this case latex) and should not be included IMO.

Reply:
So everyone can view the report without installing anything. Thought this would be easier for many (f.e. Macio, Manzke, etc)

Reply:
Then I would suggest that we publish releases by hand or via a pipeline instead. Like we do it with the source-code as well.

Reply:
@tcoro This [latex-action](https://github.com/xu-cheng/latex-action) looks quite promising. We could then simply upload it to our project website with [github-pages-deploy-action](https://github.com/JamesIves/github-pages-deploy-action). Check out our [release-app](https://github.com/project-leek/project-leek/blob/master/.github/workflows/release-app.yml) workflow as an example.

Reply:
I ❤️ DevOps!

Reply:
@tcoro You can merge the latest master now. I implemented a workflow to automatically build the report and create a docs branch.